swarm/pss/api/pssapi.go

```diff
@@ -61,8 +55,8 @@ func (pssapi *PssAPI) NewMsg(ctx context.Context, topic PssTopic) (*rpc.Subscrip
 }
 
 // SendRaw sends the message (serialized into byte slice) to a peer with topic
-func (pssapi *PssAPI) SendRaw(topic PssTopic, msg PssAPIMsg) error {
-	err := pssapi.Pss.Send(msg.Addr, topic, msg.Msg)
+func (pssapi *PssAPI) SendRaw(topic pss.PssTopic, msg pss.PssAPIMsg) error {
```

Comment:
actually no, cos it mysteriously collides with PssAPI.Pss.Send if we do that. I suggest SendPss